Testing Your Seed.md

```diff
@@ -0,0 +1,34 @@
+# Testing Your Seed
+
+Once you have made the backup of a Seed, the same can be used to access your Skywallet. However, if you have multiple Wallets you will have as many Seeds as well.
```

Comment:
I was trying to explain the scenario when a user has multiple wallets, they will have multiple Seed as well owing to each wallet having one Seed. But now I understand that it might sound confusing, so I have restructured the same. If you find it confusing still, I can scrap it.